en/docs/learn/configuring-active-directory-user-stores-for-scim-2.0-based-inbound-provisioning.md

```diff
@@ -6,7 +6,7 @@ However, when the WSO2 Identity Server is connected to an external LDAP or an Ac
 
 Add a user with the username "john" and password "Wso2@123". Here we have to map the **userName** (urn:ietf:params:scim:schemas:core:2.0:User) SCIM attribute to an existing claim in the Active Directory (e.g.: cn). Furthermore, when a user is being added in SCIM, there are four more SCIM attributes being added behind the scene. These are the SCIM attributes such as **location** (urn:ietf:params:scim:schemas:core:2.0), **created** (urn:ietf:params:scim:schemas:core:2.0), **lastModified** (urn:ietf:params:scim:schemas:core:2.0) and finally the **id** (urn:ietf:params:scim:schemas:core:2.0). So we need to map these to existing Active Directory user attributes.
 
-When mapping claims to attributes, there are a few things that need to be considered. The SCIM claim dialect (urn:ietf:params:scim:schemas:core:2.0:User and urn:ietf:params:scim:schemas:core:2.0) uses String type to hold their values. So, when mapping any SCIM claim to an attribute in the Active Directory, make sure to use the attributes which are having the String type. You can find all Active Directory attributes [here](http://www.kouti.com/tables/userattributes.htm).
+When mapping claims to attributes, there are a few things that need to be considered. The SCIM claim dialect (urn:ietf:params:scim:schemas:core:2.0:User and urn:ietf:params:scim:schemas:core:2.0) uses String type to hold their values. So, when mapping any SCIM claim to an attribute in the Active Directory, make sure to use the attributes which are having the String type. You can find all Active Directory attributes [here](https://docs.microsoft.com/en-us/windows/win32/adschema/attributes-all).
```

Comment:
`urn:ietf:params:scim:schemas:core:2.0` is this correct ? does it miss any part

Reply:
This is correct and doesn't miss any part. @AnuradhaSK please correct me if I'm wrong.